src/Writer/XLSX/Options.php

```diff
@@ -43,7 +62,13 @@ public function mergeCells(
         int $bottomRightColumn,
         int $bottomRightRow
     ): void {
+        $writer = $this->getWriter();
```

Comment:
Stateful `Options` would be confusing to use and hard to maintain.

We can simply extend `Options::mergeCells` API to have a 5th parameter for the `$sheetIndex`, defaulted to `0` to keep backward compatibility.

Reply:
Done refactoring